tests/cypress/tests/unit/formatting_test.ts

```diff
@@ -0,0 +1,129 @@
+import { getFormatter } from 'dash-table/type/number';
+
+describe('formatting', () => {
+    describe('number', () => {
+        it('returns undefined formatter with undefined format', () => {
+            const formatter = getFormatter(undefined, undefined);
+            assert.isNotOk(formatter);
+        });
+
+        describe('without nully handling / default locale', () => {
```

Comment:
A few unit tests testing mostly the wrapper behavior -- `locale + specifier + prefix + nully` with a few configurations -- as most of the functionality actually comes from d3-format